README.md

```diff
@@ -56,6 +56,8 @@ With Serena, we provide
      * Go (untested)
      * C# (untested)
      * Rust (untested)
+     * Kotlin (untested)
+     * Dart (untested)
```

Comment:
```suggestion
     * Dart (untested)
     * C++ (untested)
```

Reply:
Pushed two commits, one of which adds "C/C++" to the list (as C is also supported).